src/linkml/include_study.yaml

```diff
@@ -17,59 +17,316 @@ classes:
+  studyDescription:
+    definition_uri: include:studyDescription
+    description: Brief description of the study (2-4 sentences). Should match description in https://includedcc.org/studies.
+    title: Study Description
+    required: true
+    range: string
+  principalInvestigatorName: 
+    definition_uri: include:principalInvestigatorName
+    description: Name(s) of Principal Investigator(s) of this study; pipe-separated if multiple
+    title: Principal Investigator Name
+    required: true
     range: string
+    multivalued: true
+  studyContactName:
+    definition_uri: include:studyContactName
+    description: Name of contact person for this study; pipe-separated if multiple
+    title: Study Contact Name
+    required: true
+    range: string
+    multivalued: true
+  studyContactInstitution:
+    definition_uri: include:studyContactInstitution
+    description: Institution of contact person for this study; pipe-separated if multiple
+    title: Study Contact Institution
+    required: true
+    range: string
+    multivalued: true
+  studyContactEmail:
+    definition_uri: include:studyContactEmail
+    description: Email address of contact person for this study; pipe-separated if multiple
+    title: Study Contact Email
+    required: true
+    range: string
+    multivalued: true
   vbrEmail:
     definition_uri: include:vbrEmail
     description: Email address for Virtual Biorepository requests/inquiries
-    title: Virtual Biorepository Email
+    title: VBR Email
     range: string
   vbrUrl:
     definition_uri: include:vbrUrl
     description: Link to Virtual Biorepository request form
-    title: Virtual Biorepository URL
-    range: string
+    title: VBR URL
+    range: uri
   vbrReadme:
     definition_uri: include:vbrReadme
     description: Instructions for contacting or requesting samples from Virtual Biorepository
-    title: Virtual Biorepository Readme
+    title: VBR Readme
     range: string     
+  researchDomain:
+    definition_uri: include:researchDomain
+    description: Main research domain(s) of the study, other than Down syndrome; pipe-separated if multiple
+    title: Research Domain
+    range: enum_researchDomain
+    required: true
+    multivalued: true
+  participantLifespanStage: 
+    definition_uri: include:participantLifespanStage
+    description: Focus age group(s) of the study population; pipe-separated if multiple
+    title: Participant Lifespan Stage
+    range: enum_participantLifespanStage
+    required: true
+    multivalued: true
+  selectionCriteria:
+    definition_uri: include:selectionCriteria
+    description: Brief description of inclusion and/or exclusion criteria for the study
+    title: Selection Criteria
+    range: string
+  studyDesign:
+    definition_uri: include:studyDesign
+    description: Overall design of study, including whether it is longitudinal and whether family members/unrelated controls are also enrolled
+    title: Study Design
+    range: string 
```

Comment:
Are there any use cases to query or filter for data based on `studyDesign`?

Reply:
Not currently, but it might be nice to be able to search for longitudinal studies, or family studies. This field (and the Selection Criteria field) will probably be used more heavily for clinical trials, so there might be more opportunities to standardize when we have more trials

Reply:
Ok, from past modeling and ontology work on omics data I am more used to seeing study design as an enum to be able to filter and query data. Are the current thoughts that the range may change to an enum in the future and any past data re-annotated?

Reply:
Yes, I think several fields should be changed to enums in the future. We're still determining where to document our plans, but at the very least it should be a github issue.

Reply:
I've listed the fields that need your help in #108